README.md

```diff
@@ -74,39 +44,173 @@ This program was created with the following tech stack and packages:
-2. Download the repository
-3. Receive Kukarella files
+You can use the API locally via the following steps:
+
+1. Download and unzip the repository
+2. Install [Node.Js](https://nodejs.org/en/)
+3. Install [ffmpeg](https://ffmpeg.org/)
+   - We recommend following the steps on [adamtheautomator.com](https://adamtheautomator.com/install-ffmpeg/) for your
+   corresponding operating system. We also recommend the Powershell approach for Windows users.
+4. Add the appropriate Rhubarb files to the repository
+    1. Download [Rhubarb Lip Sync v1.13.0](https://github.com/DanielSWolf/rhubarb-lip-sync/releases/tag/v1.13.0) for
+    your corresponding operating system
+    2. Unzip and rename the folder to `rhubarb`
+    3. Move the folder into the `speaking-portal-project` directory
+    4. Delete the `extras` folder which is currently located at `speaking-portal-project\rhubarb\extras`
+5. In the terminal, navigate to `speaking-portal-project`
+6. Run `npm install` to ensure all packages are installed
+7. Run `npm start` to start the API
+8. Check in the terminal that the API is listening
+9. Create the user's Kukarella files:
     1. Go to [Kukarella](https://www.kukarella.com/)
-    2. Enter text, select a language, select a voice, and convert to audio
+    2. Enter text, select a language, select a voice, and convert the text to audio
     3. Download the audio as a .wav file
     4. Download the text file
-4. Move the Kukarella files to `speaking-portal-project\rhubarb`
-5. In the terminal, navigate to `speaking-portal-project`
-6. Install ffmpeg
-   1. Follow the steps on [adamtheautomator.com](https://adamtheautomator.com/install-ffmpeg/) for your corresponding
-   operating system. We recommend the Powershell approach for Windows users.
-7. Run `npm install`
-8. Run `npm start`
+10. Use Postman or a similar platform to send a POST request (see [POST Request Properties](#post-request-properties))
+to the local API that is currently listening for a request from step 7.
+11. Wait for the API to process the request and return the video
+12. Enjoy the animation!
 
 ## How it Works
 
-The Kurkarella TTS app creates the synthesized speech file from the user’s voice and text input. The Animation Add-On
-is broken down into two main components: the phoneme factory and the animation processor. The phoneme factory is in
-charge of mapping spoken language from an audio file into a series of phonemes, which are units of sound that
-distinguish one word from another. In order to make this process as accurate as possible, the phoneme factory also
-receives the user text file and language selection.
+### Overview
+
+The Speaking Portal Project (SPP) is built to connect with Kukurella's Text-to-Speech platform as an API. The SPP is broken down into three main components: **API**, **Phoneme Factory**, and the **Animation Factory**.
+
+### API
+
+Speaking Portal API creates an MP4 output animation from a synthesized speech file and text input based on Kukarella
```

Comment:
```suggestion
The Speaking Portal API generates an MP4 animation from a synthesized speech file and text input based on Kukarella
```
"based on kukarella TTS" feels a little weird but not sure how to fix

Reply:
what about....
```suggestion
The Speaking Portal API generates an MP4 output animation from the synthesized speech file and text input from Kukarella
```

Reply:
What exactly does synthesized in this context mean?

Reply:
created, generated, produced

Reply:
Resolving, as it was re-worded in another commit.